app/build.gradle

```diff
@@ -78,6 +78,12 @@ repositories {
     }
 }
 
+configurations.all {
+    resolutionStrategy {
+        force 'com.android.support:support-annotations:25.1.1'
```

Comment:
Can you help me understand this line? It looks like you are enforcing a specific version support-annotations for all projects and dependencies. But later you are excluding support-annotations from other dependencies. Isn't only one needed (Excluding sounds like a nice strategy)?